pkg/kubectl/cmd/version.go

```diff
@@ -74,20 +75,25 @@ func RunVersion(f cmdutil.Factory, out io.Writer, cmd *cobra.Command) error {
 		vo.ServerVersion = serverVersion
 	}
 
+	if cmdutil.GetFlagBool(cmd, "short") {
```

Comment:
Why is this logic moved around? This was part of a very long discussion. Can i get some reasoning for this?